guides/v2.2/magento-functional-testing-framework/2.2/getting-started.md

```diff
@@ -142,7 +139,7 @@ vendor/bin/robo generate:tests
 
 ### Step 7. Run tests
 
-To run tests you need a running Selenium server and a [`codecept`](commands/codeception.html) or [`robo`](commands/robo.html) with required parameters.
+To run tests, you need a running Selenium server and a [`codecept`][codecept] or a [`robo`][robo] command with required parameters.
```

Comment:
Not sure about this line, maybe it should read more like:
```
To run tests, you must run Selenium Server and execute your tests via:
* vendor/bin/mftf
* codecept
* robo
```

Reply:
MFTF 2.2 hasn't had `vendor/bin/mftf` yet.

Reply:
My mistake, didn't realize that this was in 2.2.